staging/src/github.com/kubeedge/mapper-framework/pkg/global/global.go

```diff
@@ -1,18 +1,18 @@
 package global
 
 import (
+	dmiapi "github.com/kubeedge/kubeedge/pkg/apis/dmi/v1beta1"
 	"github.com/kubeedge/mapper-framework/pkg/common"
-	"github.com/kubeedge/mapper-framework/pkg/config"
 )
 
 // DevPanel defined operations on devices, manage the lifecycle of devices
 type DevPanel interface {
 	// DevStart start device to collect/push/save data to edgecore/app/database
 	DevStart()
 	// DevInit get device info by dmi interface
-	DevInit(cfg *config.Config) error
+	DevInit(deviceList []*dmiapi.Device, deviceModelList []*dmiapi.DeviceModel) error
 	// UpdateDev update device's config and restart the device
-	UpdateDev(model *common.DeviceModel, device *common.DeviceInstance)
+	UpdateDev(model *common.DeviceModel, device *common.DeviceInstance, protocol *common.ProtocolConfig)
```

Comment:
It seems function input does not correspond to the actual call involved
```suggestion
	UpdateDev(model *common.DeviceModel, device *common.DeviceInstance)
```

![x3](https://github.com/kubeedge/kubeedge/assets/48212997/404e3089-bb55-465a-b56b-885e0eb82c94)